src/CC2Edit/ScriptTools.cpp

```diff
@@ -127,7 +127,9 @@ bool ScriptMapLoader::loadScript(const QString& filename)
                 if (!cmdNode->param() || cmdNode->param()->type() != cc2::C2GNode::NodeString)
                     continue;
                 auto paramNode = static_cast<cc2::StringNode*>(cmdNode->param());
-                QString param = QString::fromStdString(paramNode->text());
+                // Convert this to use the native separator so that C2G-referenced subdirectories properly read on systems with the / separator
+                // Luckily the / character is illegal in Windows paths, so we can do this blunt replacement without making loading some paths impossible
+                QString param = QDir::toNativeSeparators(QString::fromStdString(paramNode->text()).replace(QStringLiteral("\\"), QStringLiteral("/")));

```

Comment:
the `.replace()` bit is redundant after `QDir::toNativeSeparators`

Reply:
The `.replace()` is actually before `QDir::toNativeSeparators`, it's to convert the C2G-provided path into a Qt one and then into the native-compliant one.

Reply:
Hmm, I had thought QDir was clever enough to work without the replacement, but I just tried it, and sadly it doesn't :(.

That said, Windows (at least real Windows, not sure about WINE/Proton) does actually correctly handle forward slashes in most cases, so this might actually be unnecessary.  Have you already tried without it?

Reply:
Sorry, I'm looking again and I realize this has nothing to do with Wine/Proton, just with loading the c2g script...  That makes more sense now, sorry for the confusion.